workspaces/topology/plugins/topology/src/translations/ref.ts

```diff
@@ -0,0 +1,340 @@
+    inactive: 'Inactive',
+    updating: 'Updating',
+    evicted: 'Evicted',
+    cancelled: 'Cancelled',
+  },
+  details: {
+    name: 'Name',
+    namespace: 'Namespace',
+    labels: 'Labels',
+    annotations: 'Annotations',
+    createdAt: 'Created',
+    age: 'Age',
+    replicas: 'Replicas',
+    availableReplicas: 'Available Replicas',
+    readyReplicas: 'Ready Replicas',
+    updatedReplicas: 'Updated Replicas',
+    selector: 'Selector',
+    strategy: 'Strategy',
+    image: 'Image',
+    ports: 'Ports',
+    volumes: 'Volumes',
+    volumeMounts: 'Volume Mounts',
+    environmentVariables: 'Environment Variables',
+    resourceRequirements: 'Resource Requirements',
+    limits: 'Limits',
+    requests: 'Requests',
+    cpu: 'CPU',
+    memory: 'Memory',
+    storage: 'Storage',
+    noLabels: 'No labels',
+    noAnnotations: 'No annotations',
+    noOwner: 'No owner',
+    notAvailable: 'Not available',
+    notConfigured: 'Not configured',
+    updateStrategy: 'Update strategy',
+    maxUnavailable: 'Max unavailable',
+    maxSurge: 'Max surge',
+    progressDeadlineSeconds: 'Progress deadline seconds',
+    minReadySeconds: 'Min ready seconds',
+    desiredCompletions: 'Desired completions',
+    parallelism: 'Parallelism',
+    activeDeadlineSeconds: 'Active deadline seconds',
+    currentCount: 'Current count',
+    desiredCount: 'Desired count',
+    schedule: 'Schedule',
+    concurrencyPolicy: 'Concurrency policy',
+    startingDeadlineSeconds: 'Starting deadline seconds',
+    lastScheduleTime: 'Last schedule time',
+    maxSurgeDescription: '{{maxSurge}} greater than {{replicas}} pod',
+    maxUnavailableDescription: '{{maxUnavailable}} of {{replicas}} pod',
+  },
+  logs: {
+    download: 'Download',
+    noLogsFound: 'No logs found',
+    selectContainer: 'Select Container',
+    container: 'Container',
+    pod: 'Pod',
+    showPrevious: 'Show Previous',
+    follow: 'Follow',
+    refresh: 'Refresh',
+    timestamps: 'Timestamps',
+    wrapLines: 'Wrap Lines',
+    clearLogs: 'Clear Logs',
+    logLevel: 'Log Level',
+    search: 'Search',
+    noMatchingLogs: 'No matching logs found',
+  },
+  resources: {
+    noResourcesFound: 'No {{resourceType}} found for this resource.',
+    showingLatest: 'Showing latest {{count}} {{resourceType}}',
+  },
+  time: {
+    seconds: 'seconds',
+    minutes: 'minutes',
+    hours: 'hours',
+    days: 'days',
+  },
+  events: {
+    type: 'Type',
+    reason: 'Reason',
+    message: 'Message',
+    source: 'Source',
+    firstSeen: 'First Seen',
+    lastSeen: 'Last Seen',
+    count: 'Count',
+    noEventsFound: 'No events found',
+  },
+  filters: {
+    showLabels: 'Show Labels',
+    showPodCount: 'Show Pod Count',
+    expandApplicationGroups: 'Expand Application Groups',
+    showConnectors: 'Show Connectors',
+  },
+  common: {
+    loading: 'Loading...',
+    retry: 'Retry',
+    close: 'Close',
+    status: 'Status',
+    owner: 'Owner',
+    location: 'Location',
```

Comment:
I see just 5 `t('common....` usages. Did I miss something? Can just have these 5 for now?

<img width="735" height="770" alt="image" src="https://github.com/user-attachments/assets/7eff88ca-ccfb-43db-a41d-c0998c541bfc" />